app_flutter/.gitignore

```diff
@@ -1,5 +1,6 @@
 # Miscellaneous
 *.class
+*.lock
```

Comment:
Shouldn't we keep pubspec locks for application?

Reply:
Good catch! I didn't recognize it without pubspec in front of it